sdk/workloads/Azure.ResourceManager.Workloads/src/Generated/SapMonitorData.cs

```diff
@@ -77,5 +87,7 @@ public string ManagedResourceGroupName
         public ResourceIdentifier MonitorSubnetId { get; set; }
         /// <summary> The ARM ID of the MSI used for SAP monitoring. </summary>
         public ResourceIdentifier MsiArmId { get; }
+        /// <summary> The ARM ID of the Storage account used for SAP monitoring. </summary>
+        public string StorageAccountArmId { get; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#d4cebe383870410cab2e14d0709f78bb).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#9f6fc0ae7c8e497e98d7979dadf97a4c).

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/workloads/resource-manager/Microsoft.Workloads/preview/2022-11-01-preview/monitors.json#L1100

this is a string in swagger, so kept it same here.